webpack/scenes/ContentViews/Details/Versions/ContentViewVersionEnvironments.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label, Flex, FlexItem } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import InactiveText from '../../components/InactiveText';
+
+const ContentViewVersionEnvironments = ({ environments }) => {
+  if (environments.length === 0) {
+    return (
+      <InactiveText text={__('No environments')} />
+    );
+  }
+
+  const envLabels = environments.map(env => (
```

Comment:
```suggestion
  return environments.map(env => (
```
(and delete the return on line 25 below)